server/sonar-webserver-webapi/src/main/java/org/sonar/server/usertoken/ws/RevokeAction.java

```diff
@@ -64,6 +67,7 @@ public void handle(Request request, Response response) throws Exception {
     String name = request.mandatoryParam(PARAM_NAME);
     try (DbSession dbSession = dbClient.openSession(false)) {
       UserDto user = userTokenSupport.getUser(dbSession, request);
+      logger.info("Revoke token request by user: {} and token name: {}", user.getLogin(), name);
```

Comment:
@arunsimha-cs System Administrators can revoke any user's token.
Log message is incorrect. What we are displaying is the **user whose token is being revoked** and not the user who placed the revoke request.

<img width="1331" alt="Screenshot 2023-09-07 at 1 54 45 PM" src="https://github.com/codescan-io/sonarqube/assets/74111708/b84cf036-5822-470a-944e-03d84124807d">